src/App.jsx

```diff
@@ -38,7 +46,7 @@ function App() {
         <Route element={<PrivateRoute />}>
           <Route path="/home" element={<Home />} />
           <Route path="/:username" element={<User />} />
-          <Route path="/:username/:postId" element={<Post />} />
+          <Route path="/:username/:postId" element={<PostComments />} />
```

Comment:
suggestion: name of the component doesnt explain what the content will be. Please put a better name